dask/tests/test_distributed.py

```diff
@@ -738,6 +738,7 @@ async def test_pack_MaterializedLayer_handles_futures_in_graph_properly(c, s, a,
     assert unpacked["deps"] == {"x": {fut.key}, "y": {fut.key}, "z": {"y"}}
 
 
+@pytest.mark.filterwarnings("ignore:.*:dask.tests.warning_aliases.RemovedIn20Warning")
```

Comment:
this is protected by `pytest.importorskip("sqlalchemy")` and so should just work with:

```suggestion
@pytest.mark.filterwarnings("ignore:.*:sqlalchemy.ext.RemovedIn20Warning")
```